src/main/java/redis/clients/jedis/json/parser/RedisJsonParser.java

```diff
@@ -0,0 +1,19 @@
+package redis.clients.jedis.json.parser;
+
+/**
+ * The parser contract for all the <em>Redis Command</em> that perform JSON operations.
+ *
+ * @see redis.clients.jedis.json.RedisJsonCommands
+ * @see redis.clients.jedis.json.RedisJsonPipelineCommands
+ */
+public interface RedisJsonParser {
```

Comment:
I'm failing to see the _added benefit_ of this separate interface. Remove this.

PS: Of course we need `setJsonParser` method in necessary classes.

Reply:
That's ok. I will make the adjustment based on your suggestion. Thanks.